vpr/src/device/rr_graph_builder.h

```diff
@@ -185,15 +194,24 @@ class RRGraphBuilder {
         return node_storage_.count_rr_switches(num_arch_switches, arch_switch_inf, arch_switch_fanins);
     }
 
-    /** @brief This function reserve storage for RR nodes. */
+    /* Reserve the lists of nodes, edges, switches etc. to be memory efficient.
```

Comment:
Please follow the comment convention as other APIs, so that we are compatible to doxygen